Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -506,7 +506,7 @@ extension ARExample {
     /// - Returns: The new scene.
     private func streetsScene() -> AGSScene {
         // Create scene with the streets basemap.
-        let scene = AGSScene(basemapType: .streets)
+        let scene = AGSScene(basemapStyle: .arcGISStreets)
```

Comment:
For some reason, this basemap style won't render for me when running the sample, any ideas @mhdostal? I don't see any errors logged.

Reply:
Sometimes the scene renders _above_ you, depending on the location altitude.  Do you see the same thing with other basemap styles?

Reply:
Hm, I don't think it's that because I can get the imagery and point cloud scene to render okay. Imager does render slightly above me though like you say.

Reply:
Turns out I wasn't explicitly setting an API key, this resolves this issue.